### PR TITLE
Write data with Flux

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

**Review by kelseiv — COMMENTED** [review]
@sanderson, added some global edits in a cpl SQL database docs that will affect multiple docs across the board. Wanted to get these edits back sooner than later. Will continue editing, and do another sweep when you're ready.

File: content/flux/v0.x/write-data/sql/_index.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+- `mysql`
+- `postgres`
+- `snowflake`
+- `sqlite3`
+- `sqlserver`, `mssql`
+
+## Data source names
+Each [SQL driver](#drivers) supports unique data source name (DSN) syntaxes
+(also known as **connection strings**).
+_See the [database guides](#databases) for information about DSNs for each driver._
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/flux/v0.x/write-data/sql/_index.md:58]: This doc does represent the "Overview" of SQL connections.

File: content/flux/v0.x/write-data/sql/_index.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+username = secrets.get(key: "POSTGRES_USER")
+password = secrets.get(key: "POSTGRES_PASS")
+
+sql.to(
+  driverName: "postgres",
+  dataSourceName: "postgresql://${username}:${password}@localhost:5432",
+  table: "example_table"
+)
+```
+
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/flux/v0.x/write-data/sql/_index.md:79]: That's the role that this page plays. It's a landing page for all the SQL docs and contains high-level conceptual information for writing to SQL databases with Flux.

File: content/flux/v0.x/write-data/sql/amazon-rds.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+- [Write to SQL Server](/flux/v0.x/write-data/sql/sql-server/)
+
+## Amazon RDS connection credentials
+Amazon RDS provides connection credentials required to connect to your database instance.
+The following links provide more information for each database engine:
+
+- [Connect to MariaDB](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ConnectToMariaDBInstance.html)
+- [Connect to MySQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ConnectToInstance.html)
+- [Connect to PostgreSQL](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ConnectToPostgreSQLInstance.html)
+- [Connect to SQL Server](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_ConnectToMicrosoftSQLServerInstance.html)
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/flux/v0.x/write-data/sql/amazon-rds.md:68]: Amazon RDS is a service that provides managed instances for each of these databases. They provide instructions for how to retrieve connection credentials specific to RDS. That's what these links are linking to. The other pages address using OSS or "unmanaged" instances of each.

File: content/flux/v0.x/write-data/influxdb.md [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+import "experimental"
+
+data
+  |> experimental.to(
+    bucket: "example-bucket",
+    org: "example-org",
+    token: "mY5uPeRs3Cre7tok3N",
+    host: "https://myinfluxdbdomain.com/8086"
+  )
+```
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/flux/v0.x/write-data/influxdb.md:205]: The length of the table labels would break tabs.

**Pull request merged**